docs/content/reference/server-configuration.md

```diff
@@ -1,52 +1,208 @@
 ---
 title: "Server configuration"
-draft: true
-pre: '<i class="fas fa-cogs"></i> '
+draft: false
+description: "Reference material for the server_config.json file"
+pre: "<i class='fa fa-cogs'></i> "
 ---
+{{< table_of_contents >}}
 
-## Configuring the Monkey Island
+## Configuration
 
-The Monkey Island Server is configured through the `server_config.json` file.
+<!--
+This documentation was autogenerated by passing the plugin's config-schema.json
+through https://github.com/adobe/jsonschema2md. It was then modified by hand to
+remove extraneous information.
+-->
 
-{{% notice info %}}
-Refer to the [setup guides](../../setup/) to learn how to use
-the `server_config.json` file for each deployment.
-{{% /notice %}}
+| Property                             | Type      | Required | Nullable       |
+| :----------------------------------- | :-------- | :------- | :------------- |
+| [data\_dir](#data_dir)               | `string`  | Optional | cannot be null |
+| [log\_level](#log_level)             | `string`  | Optional | cannot be null |
+| [mongodb](#mongodb)                  | `object`  | Optional | cannot be null |
+| [ssl\_certificate](#ssl_certificate) | `object`  | Optional | cannot be null |
+| [island\_port](#island_port)         | `integer` | Optional | cannot be null |
 
-### Creating a configuration file
+### data\_dir
+
+The directory where the Island will store runtime artifacts.
+
+`data_dir`
+
+* is optional
+
+* Type: `string`
+
+* cannot be null
+
+#### data\_dir Constraints
+
+The value of this string must be a valid path.
+
+#### data\_dir Default Value
+
+The default value for Windows is:
+
+```json
+"%AppData%\\monkey_island"
+```
+
+The default value for Linux is:
+
+```json
+"$HOME/.monkey_island"
+```
+
+### log\_level
+
+The threshold for the Island logger.
+
+`log_level`
+
+* is optional
+
+* Type: `string`
```

Comment:
Should we list the acceptable values?

Reply:
We don't have any constraints on the value in the code so it doesn't make sense to add it ourselves in the reference documentation.

Reply:
It might be worth adding the constraints in the code.